scripts/check-single-extension.js

```diff
@@ -43,6 +43,25 @@ exports.verifyExtension = async function (extensionName, options) {
     return { code: 'invalid-json' };
   }
 
+  // Basic check to see if it is a GDevelop project
+  if (
+    typeof extension.properties === 'object' &&
+    typeof extension.properties.name === 'string'
+  ) {
+    return { code: 'gdevelop-project-file' };
+  }
+
+  // Basic check to see if it is a GDevelop extension
+  if (
+    !(
+      Array.isArray(extension.eventsFunctions) &&
+      Array.isArray(extension.eventsBasedBehaviors) &&
+      typeof extension.name === 'string'
+    )
```

Comment:
This check might be a bit fragile? If in we stop storing empty arrays for example if an extension only has a behavior and no free functions, a legit GDevelop extension might come out not being detected as a GDevelop extension.